Mathlib/CategoryTheory/ConcreteCategory/Basic.lean

```diff
@@ -107,7 +107,6 @@ def ConcreteCategory.hasCoeToFun {X Y : C} : CoeFun (X ⟶ Y) fun _ => X → Y :
 attribute [local instance] ConcreteCategory.hasCoeToFun
 
 /-- In any concrete category, we can test equality of morphisms by pointwise evaluations.-/
-@[ext]
```

Comment:
Maybe this should just be lower priority instead of removing the attribute? You can write something like `@[ext 900]`, the default is `1000`

Reply:
:+1: